google-cloud-bigquerystorage/src/main/java/com/google/cloud/bigquery/storage/v1/Exceptions.java

```diff
@@ -94,23 +103,34 @@ protected SchemaMismatchedException(Status grpcStatus, String name) {
     }
   }
 
-  /** Offset already exists. */
+  /**
+   * Offset already exists. This happens when appending with offset that is less than the current
```

Comment:
Offset already exists.  This indicates that the append request attempted to write data to an offset before the current end of the stream.  To append data successfully, you must either specify the offset corresponding to the current end of stream, or omit the offset from the append request.